ddtrace/version.py

```diff
@@ -5,12 +5,5 @@ def get_version():
 
         return version
     except ImportError:
-        import pkg_resources
-
-        try:
-            # something went wrong while creating _version.py, let's fallback to pkg_resources
-
-            return pkg_resources.get_distribution(__name__).version
-        except pkg_resources.DistributionNotFound:
-            # package is not installed
-            return "dev"
```

Comment:
try:
            from importlib.metadata import version
        except ImportError:
            # required for python3.7
            from importlib_metadata import version  # type: ignore[no-redef]

        try:
            version(__name__)
        except pkg_resources.DistributionNotFound:
           # package is not installed
           return "dev"